code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -722,3 +708,10 @@
 ///This proc is used for adding the swabbale element to mobs so that they are able to be biopsied and making sure holograpic and butter-based creatures don't yield viable cells samples.
 /mob/living/simple_animal/proc/add_cell_sample()
 	return
+
+/mob/living/simple_animal/relaymove(mob/living/user, direction)
+	if(user.incapacitated())
+		return
+	if(SEND_SIGNAL(src, COMSIG_RIDDEN_DRIVER_MOVE, user, direction) & COMPONENT_DRIVER_BLOCK_MOVE)
```

Comment:
Can this be refactored to be sent at the base of the proc?

Reply:
Actually, this signal is being sent from two different sources.
Signals should be unique, reporting a single event.
And at the base of the proc if possible.